cql-engine-fhir/src/test/java/org/opencds/cqf/cql/data/fhir/FhirExecutionTestBase.java

```diff
@@ -65,11 +64,10 @@ public void setup() {
         dstu3Context.newRestfulGenericClient("http://measure.eval.kanvix.com/cqf-ruler/baseDstu3"));
         dstu3Provider = new CompositeDataProvider(dstu3ModelResolver, dstu3RetrieveProvider);
         
-        hl7ModelResolver = new HL7FhirModelResolver();
```

Comment:
Same as above. This section of the tests specific to the hl7 resolver can go away.